From 7b4905d10fa44d76e25e8c02ea0aa526f933f977 Mon Sep 17 00:00:00 2001
From: Cristian Birsan <cristian.birsan@microchip.com>
Date: Wed, 23 Feb 2022 23:20:03 +0200
Subject: [PATCH 430/471] ARM: configs: at91: sama7: add sama7 USB phy

Add CONFIG_PHY_MICROCHIP_SAMA7_USB. It is needed to use USB Host
or Device on sama7.

Signed-off-by: Cristian Birsan <cristian.birsan@microchip.com>
---
 arch/arm/configs/sama7_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/sama7_defconfig b/arch/arm/configs/sama7_defconfig
index ecfd71900a39..928c03fc8916 100644
--- a/arch/arm/configs/sama7_defconfig
+++ b/arch/arm/configs/sama7_defconfig
@@ -199,6 +199,7 @@ CONFIG_PWM=y
 CONFIG_PWM_ATMEL=y
 CONFIG_MCHP_EIC=y
 CONFIG_RESET_CONTROLLER=y
+CONFIG_PHY_MICROCHIP_SAMA7_USB=y
 CONFIG_EXT2_FS=y
 CONFIG_EXT3_FS=y
 CONFIG_FANOTIFY=y
-- 
2.34.1

